### DISCUSSION TITLE
Avatar Head Bobbing

### DISCUSSION THREAD

**Discussion by Kanishk-Malick** [mozilla/hubs#2818] [General Questions & Discussion] [answered]
I uploaded my avatar on Hubs, but whenever I speak, I have this issue of the whole head bobbing instead of lips moving. How can I solve this? I used avatarsdk.com for creating avatar and modelconverter.com to convert it to obj.

**Comment by cvanmeurs** [comment, marked as answer]
I'm not familiar with that particular avatar generator, but it sounds like your avatar might be using the "scale audio feedback" component instead of the "morph audio feedback" component.  Jim and Dom did a great tutorial video/stream just a week or so ago that covered this in detail.  The video link is: https://www.youtube.com/watch?v=qBvZhh6KVcg

The best way to author a custom avatar to Hubs is by using our Hubs component extension for Blender, which you can download here:  https://github.com/MozillaReality/hubs-blender-exporter

And you can find Blender for free here:  https://www.blender.org/

I believe the latest version of Blender that our Hubs component add-on supports is 2.83 right now.

**Reply by johnshaughnessy** [reply]
Marking this as the answer. Ping me if this isn't satisfactory @Kanishk-Malick
